istioctl/pkg/admin/istiodconfig.go

```diff
@@ -371,7 +371,7 @@ func (c *ControlzClient) GetScope(scope string) (*ScopeInfo, error) {
 var (
 	istiodLabelSelector = ""
 	istiodReset         = false
-	validationPattern   = `^[\w ]+:(debug|error|warn|info|debug)`
+	validationPattern   = `^[\w\- ]+:(debug|error|warn|info|debug)`
```

Comment:
```suggestion
	validationPattern   = `^[\w\- ]+:(error|warn|info|debug)`
```

Reply:
After checking again, I found that there should be a `none`